modules/ibmz-configure-nbde-with-static-ip.adoc

```diff
@@ -120,35 +120,36 @@ ifndef::ibm-z-kvm[]
 ----
 rd.neednet=1 \
 console=ttysclp0 \
-coreos.inst.install_dev=/dev/dasda \ <1>
+coreos.inst.install_dev=/dev/dasda \// <1>
 ignition.firstboot ignition.platform.id=metal \
-coreos.live.rootfs_url=http://10.19.17.25/redhat/ocp/rhcos-413.86.202302201445-0/rhcos-413.86.202302201445-0-live-rootfs.s390x.img \
-coreos.inst.ignition_url=http://bastion.ocp-cluster1.example.com:8080/ignition/master.ign \
+coreos.live.rootfs_url=<rhcos_liveos> \// <2>
+coreos.inst.ignition_url=<rhcos_ign> \// <3>
```

Comment:
The value requires an url. "rhcos_ign" is used as ignition filename.
Therefore, I prefer to use here: "rhcos_ign_url"
```suggestion
coreos.inst.ignition_url=<rhcos_ign_url> \// <3>
```